MAUI/TimePicker/customizations.md

```diff
@@ -0,0 +1,403 @@
+
+{% tabs %}
+
+{% highlight xaml tabtitle="XAML" %}
+
+<picker:SfTimePicker x:Name="Timepicker" >
+    <picker:SfTimePicker.ColumnHeaderView >
+        <picker:TimePickerColumnHeaderView HourHeaderText="Hour Column"
+                                           MinuteHeaderText="Minute Column"
+                                           SecondHeaderText="Second Column"
+                                           MeridiemHeaderText="Meridiem Column"/>
+    </picker:SfTimePicker.ColumnHeaderView>
+</picker:SfTimePicker>
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="C#" %}
+
+SfTimePicker timePicker = new SfTimePicker();
+timePicker.ColumnHeaderView = new TimePickerColumnHeaderView()
+{
+    HourHeaderText = "Hour Column",
+    MinuteHeaderText = "Minute Column",
+    SecondHeaderText = "Second Column",
+    MerdiemHeaderText = "Merdiem Column"
+};
+
+this.Content = timePicker;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Set divider color
+
+The SfTimePicker control allows you to customize the column header divider color by setting the [DividerColor] property of [TimePickerColumnHeaderView].
+
+{% tabs %}
+
+{% highlight xaml tabtitle="XAML" hl_lines="3" %}
+
+<picker:SfTimePicker x:Name="Timepicker" >
+    <picker:SfTimePicker.ColumnHeaderView >
+        <picker:TimePickerColumnHeaderView DividerColor="#A9A9A9" />
+    </picker:SfTimePicker.ColumnHeaderView>
+</picker:SfTimePicker>
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="C#" %}
+
+SfTimePicker timePicker = new SfTimePicker();
+timepicker.ColumnHeaderView = new TimePickerColumnHeaderView()
+{
+    DividerColor = Color.FromArgb("#A9A9A9")
+};
+
+this.Content = timePicker;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Customization of column header
+
+You can customize the column header view text style and background color of the `Time Picker` using the [TextStyle] and [Background] properties of the [TimePickerColumnHeaderView].
+
+{% tabs %}
+
+{% highlight xaml tabtitle="XAML" %}
+
+<picker:SfTimePicker x:Name="Timepicker" >
+    <picker:SfTimePicker.ColumnHeaderView >
+        <picker:TimePickerColumnHeaderView Background="#6750A4">
+            <picker:TimePickerColumnHeaderView.TextStyle >
+                <picker:PickerTextStyle FontSize="15" TextColor="White" />
+            </picker:TimePickerColumnHeaderView.TextStyle>
+        </picker:TimePickerColumnHeaderView>
+    </picker:SfTimePicker.ColumnHeaderView>
+</picker:SfTimePicker>
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="C#" %}
+
+SfTimePicker timePicker = new SfTimePicker();
+timePicker.ColumnHeaderView = new TimePickerColumnHeaderView()
+{
+    Background = Color.FromArgb("#6750A4"),
+    TextStyle = new PickerTextStyle()
+    {
+        TextColor = Colors.White,
+        FontSize = 15,
+    }
+};
+
+this.Content = timePicker;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Footer customization
+
+You can customize the time picker footer view by using the `FooterView` property of `SfTimePicker`.
```

Comment:
Customize the time picker footer view by using the `FooterView` property of the `SfTimePicker`.